lib/Imagine/Gd/Drawer.php

```diff
@@ -100,12 +127,26 @@ public function chord(PointInterface $center, BoxInterface $size, $start, $end,
         return $this;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function circle(PointInterface $center, $radius, ColorInterface $color, $fill = false, $thickness = 1)
+    {
+    	// GD do not have a circle function, ellipse is used:
+    	return $this->ellipse($center, new Box($radius, $radius), $color, $fill, $thickness);
+    }
+
     /**
      * {@inheritdoc}
      */
     public function ellipse(PointInterface $center, BoxInterface $size, ColorInterface $color, $fill = false, $thickness = 1)
     {
-        imagesetthickness($this->resource, max(1, (int) $thickness));
+    	if ((0 === ($thickness = max(1, (int) $thickness))) && (!$fill)) {
```

Comment:
`$thickness` will always be greater than 0 because of the `max` call...

Reply:
What about removing unnecessary parenthesis?
```php
if (0 === ($thickness = max(1, (int) $thickness)) && !$fill) {
```

PS: not a requirement, but for a better readability I'd prefer to split the assignment and the `if` in two separate lines:
```php
$thickness = (int) $thickness;
if (0 === $thickness && !$fill) {
```